### ISSUE TITLE
Fail!

### ISSUE THREAD

**Issue by Nemodong** [Earthcomputer/EnchantmentCracker#164]
Hi, sorry about this message being another Fail message. However, i am very very sure that i copied every step and did everything i could possibly do to prevent any warnings. Yes, I have seen the list of what not to dos. I unequipped everything i have on me. However, after i calculate the two seeds, the final seed shows Failed? I do not know what to do anymore. I am on a 1.15.2 minecraft server. And yes i have tried it myself in a normal single player game, but it still did not work. Would you please help, thanks a lot.

**Comment by Earthcomputer** [comment]
?template failed

**Comment by Earthcomputer** [comment]
Specifically for your case I would recommend trying out the mod specifically to tell you exactly which step you're missing.

**Comment by Nemodong** [comment]
would you please go over the steps since im not great w java? sorry for the convience

**Comment by Nemodong** [comment]
and thx for the such fast reply

**Comment by Earthcomputer** [comment]
You shouldn't need to understand Java, just install Fabric loader, and drop clientcommands and fabric API into your mods folder.

**Comment by Nemodong** [comment]
or if u have time and hope this is not too huge of a task, can you maybe jus join the server and see?

**Comment by Earthcomputer** [comment]
I don't have enough time for that, I'm sorry.

**Comment by Nemodong** [comment]
okok thats fine.

**Comment by Nemodong** [comment]
<img width="619" alt="Screen Shot 2020-05-09 at 8 19 42 PM" src="https://user-images.githubusercontent.com/65073533/81473649-7bb98f80-9232-11ea-93a5-801de0c720fd.png">

**Comment by Nemodong** [comment]
do you know which one i open

**Comment by Earthcomputer** [comment]
Just download and run the installer jar from https://fabricmc.net/use/

**Comment by Nemodong** [comment]
um would you please send the link for the mod tutorial u have posted?

**Comment by Nemodong** [comment]
nvm found it sorry

**Comment by Nemodong** [comment]
<img width="700" alt="Screen Shot 2020-05-09 at 8 25 00 PM" src="https://user-images.githubusercontent.com/65073533/81473744-29c53980-9233-11ea-9a56-67b425793b94.png">

**Comment by Nemodong** [comment]
i have this^

**Comment by Earthcomputer** [comment]
Yes that looks good, just press install

**Comment by Nemodong** [comment]
ok thx

**Comment by Nemodong** [comment]
and then what do i do?

**Comment by Nemodong** [comment]
<img width="928" alt="Screen Shot 2020-05-09 at 8 27 22 PM" src="https://user-images.githubusercontent.com/65073533/81473778-7f99e180-9233-11ea-9287-7d95b05ba9e9.png">

**Comment by Earthcomputer** [comment]
You need to download the clientcommands jar from https://github.com/Earthcomputer/clientcommands/releases (linked above), and the fabric API jar from curseforge (also linked above). Move these files into the `/Users/nemo2019/Library/Application Support/minecraft/mods` folder which should have been created.

**Comment by Nemodong** [comment]
thx a lot, wait for a second

**Comment by Nemodong** [comment]
<img width="601" alt="Screen Shot 2020-05-09 at 8 31 02 PM" src="https://user-images.githubusercontent.com/65073533/81473843-0222a100-9234-11ea-9efb-956d4baceecb.png">

**Comment by Nemodong** [comment]
in the link, theres only this^

**Comment by Earthcomputer** [comment]
That looks like the correct clientcommands jar, looks like you've also done this before for KaptainWutax's mod. Make sure to also install the fabric API jar from curseforge (which I see you were on a minute ago)

**Comment by Nemodong** [comment]
haha ya using the ender pillars to figure out the seed

**Comment by Nemodong** [comment]
in a server^ cuz the admin dont wanna tell me

**Comment by Nemodong** [comment]
<img width="905" alt="Screen Shot 2020-05-09 at 8 34 24 PM" src="https://user-images.githubusercontent.com/65073533/81473901-7bba8f00-9234-11ea-808c-59e11e7864e3.png">

**Comment by Nemodong** [comment]
so i download this? just to make sure

**Comment by Earthcomputer** [comment]
Yes that's the one

**Comment by Nemodong** [comment]
kk thx

**Comment by Nemodong** [comment]
ok i have both now

**Comment by Nemodong** [comment]
<img width="613" alt="Screen Shot 2020-05-09 at 8 36 44 PM" src="https://user-images.githubusercontent.com/65073533/81473948-cd631980-9234-11ea-8ad8-a48738fbe2b2.png">

**Comment by Nemodong** [comment]
thx for helping me out btw, know this is a bit annoying

**Comment by Earthcomputer** [comment]
Now run the game using the new launcher profile that the fabric installer created, and try to crack the player seed in a single-player world, following the instructions that the bot gave you earlier. The mod should tell you if you do something wrong.

**Comment by Nemodong** [comment]
ok thank you

**Comment by Nemodong** [comment]
does the seed change or it always stay the same

**Comment by Earthcomputer** [comment]
It stays the same as long as you don't do anything from the list of disallowed actions (which includes logging out). If it changes, you have to re-crack it again.

**Comment by Nemodong** [comment]
ok got it thanks

**Comment by Nemodong** [comment]
<img width="353" alt="Screen Shot 2020-05-09 at 8 45 10 PM" src="https://user-images.githubusercontent.com/65073533/81474131-fb952900-9235-11ea-9d5a-366d998317fa.png">

**Comment by Nemodong** [comment]
do i paste this into the enchcracker?

**Comment by Earthcomputer** [comment]
You shouldn't need the external enchantment cracker, this mod can do everything it does (and more). To actually manipulate enchantments, once you've cracked the seed, try out the `/cenchant` command.

**Comment by Nemodong** [comment]
ah i see

**Comment by Nemodong** [comment]
i think it worked on the single player, lemme try the server

**Comment by Nemodong** [comment]
its so much faster than the enchcracker

**Comment by Earthcomputer** [comment]
Btw, the mod can crack the seed much quicker by using the exact velocity of items you throw out. Make sure you have enough items in your inventory, then run `/ccrackrng`.

**Comment by Nemodong** [comment]
sorry but i still dont get how to combine the two seeds in the mod

**Comment by Nemodong** [comment]
and thanks for that tip

**Comment by Earthcomputer** [comment]
Iirc, there is a button in the right hand side of the enchanting table GUI to combine the two seeds. Or maybe I am misremembering and it combines them automatically. Anyway, if you're only interested in the mod, you should be able to run `/ccrackrng` then straight away `/cenchant`, no combining of any seeds needed.

**Comment by Nemodong** [comment]
got it

**Comment by Nemodong** [comment]
<img width="673" alt="Screen Shot 2020-05-09 at 8 57 32 PM" src="https://user-images.githubusercontent.com/65073533/81474391-b5d96000-9237-11ea-8a97-dc813090cc68.png">

**Comment by Nemodong** [comment]
<img width="364" alt="Screen Shot 2020-05-09 at 8 59 18 PM" src="https://user-images.githubusercontent.com/65073533/81474421-f507b100-9237-11ea-8835-dfae9a19d888.png">

**Comment by Earthcomputer** [comment]
Did you do multiple dummy enchantments or something? It should have given a message in chat saying what you did wrong.

**Comment by Nemodong** [comment]
so this means i have cracked it?

**Comment by Nemodong** [comment]
nono it didnt send me any message

**Comment by Earthcomputer** [comment]
Player crack state should be "cracked"

**Comment by Nemodong** [comment]
um

**Comment by Nemodong** [comment]
let me try again

**Comment by Nemodong** [comment]
<img width="665" alt="Screen Shot 2020-05-09 at 9 02 25 PM" src="https://user-images.githubusercontent.com/65073533/81474495-63e50a00-9238-11ea-8eba-028c1ad74059.png">

**Comment by Nemodong** [comment]
so this is the first step right

**Comment by Nemodong** [comment]
and i need to enchant a dummy again in order to crack the second one right

**Comment by Earthcomputer** [comment]
Yes. I hope you remember to enchant a dummy before the first step as well.

**Comment by Nemodong** [comment]
yes yes i did that

**Comment by Nemodong** [comment]
and when i cracked the second one
the player crack state shows uncracked

**Comment by Nemodong** [comment]
maybe that explains why it always shows "failed!"?

**Comment by Earthcomputer** [comment]
Probably. Try running `/ctemprule set playerRNGMaintenance false` to turn off auto-recovery from errors, which might be messing up with the process.

**Comment by Nemodong** [comment]
<img width="492" alt="Screen Shot 2020-05-09 at 9 08 18 PM" src="https://user-images.githubusercontent.com/65073533/81474603-364c9080-9239-11ea-8da8-5d8a37c10065.png">

**Comment by Nemodong** [comment]
um nope still dont think its working?

**Comment by Nemodong** [comment]
cuz when i did this in the single player, i combined those two in the enchcracker, and it gave the final seed. however for this, i put those two in the enchcracker, it shows fail

**Comment by Earthcomputer** [comment]
Do you get a warning about a modded server when you join?

**Comment by Nemodong** [comment]
um hol on lemme check

**Comment by Nemodong** [comment]
for single player when it shows cracked

**Comment by Nemodong** [comment]
<img width="550" alt="Screen Shot 2020-05-09 at 9 12 36 PM" src="https://user-images.githubusercontent.com/65073533/81474713-d0143d80-9239-11ea-98e9-50e8212d2ea0.png">

**Comment by Nemodong** [comment]
where do u the final seed

**Comment by Earthcomputer** [comment]
Once it's cracked you can run `/cenchant`. But for multiplayer, do you get a warning about a modded server when you log in?

**Comment by Nemodong** [comment]
oh no it does say the server is modded

**Comment by Nemodong** [comment]
is it because of the tnt being banned

**Comment by Earthcomputer** [comment]
?template modded

**Comment by Nemodong** [comment]
or is it cuz of the seed cracker?

**Comment by Earthcomputer** [comment]
It's not because of the seed cracker, it's probably because PaperMC is installed on the server. It's not that I won't support modded servers, it's that I actually can't, there is no way.

**Comment by Nemodong** [comment]
um are u leaving?

**Comment by Nemodong** [comment]
thx

**Comment by Nemodong** [comment]
ok i see sorry for ur time
